api/models.py

```diff
@@ -143,7 +144,8 @@ class Society(Base):
     _total_points = db.Column(db.Integer, default=0)
 
     members = db.relationship('User', backref='society', lazy='dynamic')
-    logged_activities = db.relationship('LoggedActivity', backref='society')
+    logged_activities = db.relationship('LoggedActivity', backref='society',
```

Comment:
@LarryWachira The backref on this relationship seems to be off, Is this the intended relation?

Reply:
Elaborate. The relationship is well defined and works, as far as I know.

Reply:
To be specific isn't the naming rather misleading? Given that the attribute is `logged_activities` then one might expect that the backref would have some relation to logged activities i.e `society_logged_activities` rather than `society`.

Reply:
Alright, I'll get to it. `participating_society` would fit in much better. And yes, the backref is meant to represent the society the activity was logged for.